ivy_tests/test_ivy/test_functional/test_nn/test_activations.py

```diff
@@ -136,14 +136,19 @@ def test_log_softmax(*, dtype_and_x, axis, test_flags, backend_fw, fn_name, on_d
 @handle_test(
     fn_tree="functional.ivy.mish",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        large_abs_safety_factor=8,
-        small_abs_safety_factor=8,
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        large_abs_safety_factor=12,
+        small_abs_safety_factor=12,
+        min_dim_size=2,
         safety_factor_scale="log",
     ),
+    ground_truth_backend="jax",
 )
 def test_mish(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
     dtype, x = dtype_and_x
+    x = x[0]
+    if len(x.shape) < 2:
+        x = x.reshape(-1, 1)
```

Comment:
Why is this needed? Calling `ivy.mish` on a 1D or even 0D array shouldn't cause any problems

Reply:
it's something related also to paddle backend when I extract the real and imag part of the complex input it transforms them to **0D** although the input is 1D dimension array.

Reply:
Isn't this still doing `x[0]` afterwards, so if the input was 1D then it's still getting a 1D array but only with one element

Reply:
x is initially a list of arrays I took the first array from the list and expand it's dims.

Reply:
Yes, but the test still does `x[0]` afterwards, so for an input of say `x = [ivy.array([1,2,3])]` this process turns it into `x = ivy.array([[1], [2], [3]])` and then the input to `test_function` is `x = x[0] = ivy.array([1])`. I also tried removing this section and re-running it locally and the tests still passed